packages/pg/test/unit/connection-parameters/creation-tests.js

```diff
@@ -172,6 +172,26 @@ test('libpq connection string building', function () {
     )
   })
 
+  test('builds conn string with options', function () {
```

Comment:
What happens if both `options` and `statement_timeout` is included in the config?

Reply:
ah! thats good catch, it would overridden the second the second `options=` attr. Updated the logic and added some more test.